### PR TITLE
docs: grpc vs graphql

### LINKED ISSUE #321: Write an article for gRPC vs GraphQL
Write an article for GraphQL vs gRPC: 
The outline and some ground work is done in [this PR](#320).

## Content Creation Requirements

To maintain the quality of our content, please adhere to the following guidelines:

### 1. Accuracy in Language

- **Grammar and Spelling:** Ensure your content is free from grammatical and spelling errors.

### 2. Tone and Style

- **Neutral Tone:** Maintain a neutral, non-emotional tone throughout the content.
- **Engaging Style:** Write in a free-flowing and engaging manner, keeping the reader's interest.

### 3. Content Integrity

- **Fact-Checking:** Verify all information. If unsure, use Discord to clarify.
- **Relevance:** Ensure all content is cohesive, to the point, and directly related to the title.
- **Performance Claims:** Substantiate any performance-related statements with benchmark data. Include relevant links or appendices showcasing these benchmarks to maintain credibility.
- **Code Verification:** Ensure all code snippets included in the article are linked to a working GitHub repository.

### 4. Originality

- **Avoid Low-Effort Content:** Content should be original and not solely generated by AI tools like ChatGPT.
- **Image Usage:** Use only images that are free from copyright restrictions or for which you have the appropriate rights.

### Expected Word count: 1800

**PS: Adherence to these guidelines is crucial. Content not meeting these standards may require revision or may not be accepted.**

### PR DESCRIPTION
**Pull request by asr2003**
Fixes  #321
/claim #321 

References taken :

https://blog.postman.com/grpc-vs-graphql/

https://medium.com/@svengau/when-graphql-meets-grpc-3e9729d32e05

### CODE REVIEW

**Comment by asr2003** [comment]
Please review @amitksingh1490 . Any changes needed I will incorporate them.

**Comment by amitksingh1490** [comment]
image should not be copied from other websites. There are todos pending

File: graphql/graphql-vs-grpc.mdx [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+}
+
+message HelloReply {
+  string message = 1;
+}
+```
+These `.proto` files are then compiled into client and server code in the language of your choice, allowing you to easily integrate gRPC into your existing codebase.
+
+The framework also leverages HTTP/2 for transport, ensuring efficient, multiplexed communication between clients and servers.
+   #### TODO: Put a diagram explaining How it works here
```
Comment (amitksingh1490) [inline graphql/graphql-vs-grpc.mdx:56]: where is this?

Comment (asr2003) [reply graphql/graphql-vs-grpc.mdx:56]: Maybe i missed it out :)

**Comment by asr2003** [comment]
> image should not be copied from other websites. There are todos pending

@amitksingh1490 I think we can use non-copyright images right? Then what is to do further for this?

**Comment by amitksingh1490** [comment]
Closing the PR as copying content or images from other websites is prohibited. When writing the article, please read the research and convey your own thoughts or experiences instead of rewriting someone else's ideas.

Please review the Content Creation Requirements. This is the first warning. Next time, we will bar you from contributing.

**Comment by asr2003** [comment]
@amitksingh1490 Is anything wrong with this PR? Any suggestions on how to improve on this? I just used their articles as reference

**Comment by amitksingh1490** [comment]
4. Originality
Avoid Low-Effort Content: Content should be original and not solely generated by AI tools like ChatGPT.

**Comment by asr2003** [comment]
@amitksingh1490 How images can be integrated? Does it need to draw new images using canva maybe?

**Comment by amitksingh1490** [comment]
@asr2003  You can use https://excalidraw.com/ canva or any other tool of your choice but please avoid using images from other blogposts without giving them credits

**Comment by asr2003** [comment]
OK! I will follow this guidelines. What is your suggested way of using images creating a new images or using images by giving credits?

**Comment by amitksingh1490** [comment]
creating a new image in this case. By creating new I don't mean replicate the same image. Be creative.

**Pull request closed without merging**